doc/conf.py

```diff
@@ -59,7 +60,8 @@
 
 # General information about the project.
 project = u'benchopt'
-copyright = u'2020-2022, Benchopt contributors'
+from datetime import datetime  # noqa
```

Comment:
does this deserve a noqa? it could go at the top of the file

Reply:
yes indeed!